include/openssl/nid.h

```diff
@@ -4290,6 +4290,9 @@ extern "C" {
 #define SN_KYBER512 "KYBER512"
```

Comment:
Delete this?

Reply:
can't before deleting the obsolete KEM implementation, which I can't delete before we switch to the new API two projects that are already using the obsolete implementation :)

Reply:
Touche 🗡️